plugin/src/test/sc/plugin2021/xstream/ConverterTest.kt

```diff
@@ -143,6 +143,12 @@ class ConverterTest: WordSpec({
                         <shape>PENTO_Y</shape>
                       </greenShapes>
                       <lastMoveMono class="linked-hash-map"/>
+                      <validColors class="linked-hash-set">
```

Comment:
these class tags shouldn't be here!

Reply:
do you know of a way to remove them?

Reply:
wasn't there an annotation like `@XStreamIntrinsic` for this? We serialize collections in other places, so we must have the solution already.

Reply:
There is `@XStreamImplicit` which removes the tag entirely.
That's what's being used elsewhere for lists.
However, if we use that for game states we put all tags from all the lists (undeployedPieceShapes, validColors, LastMoveMono) on the same level. That works for the relatively small lists like `players' in the RoundBasedGameInstance, but for GameState it'd defeat the purpose of XML.
Also, it'd mean changing the XML specification notably, so the best solution is probably to write a custom converter